app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/recycleradapters/AllReviewsAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package org.fossasia.susi.ai.skills.feedback.adapters.recycleradapters
+
+import android.content.Context
+import android.support.annotation.NonNull
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import org.fossasia.susi.ai.R
+import org.fossasia.susi.ai.rest.responses.susi.Feedback
+import org.fossasia.susi.ai.skills.feedback.adapters.viewholders.AllReviewsViewHolder
+
+/**
+ *
+ * Created by arundhati24 on 27/06/2018
+ */
+class AllReviewsAdapter(val context: Context, val feedbackList: List<Feedback>?) :
+        RecyclerView.Adapter<AllReviewsViewHolder>() {
+
+    @NonNull
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): AllReviewsViewHolder {
+        val itemView = LayoutInflater.from(parent.context)
+                .inflate(R.layout.item_feedback, parent, false)
+        return AllReviewsViewHolder(itemView)
+    }
+
+    override fun getItemCount(): Int {
+        if (feedbackList != null) {
+            return feedbackList.size
+        }
+        return 0
+    }
+
+    @NonNull
+    override fun onBindViewHolder(holder: AllReviewsViewHolder, position: Int) {
+        if (feedbackList != null) {
+            if (feedbackList[position] != null) {
+                if (feedbackList[position].email != null) {
```

Comment:
TextUtils isEmpty

Reply:
We will have to check for nullability here but I am not sure if it can be an empty String ever. 
But, okay for safety I am adding a chech for empty string as well.